test/queries/test_asset.py

```diff
@@ -1,15 +1,15 @@
 """
     Tests assets query.
 """
-import os
 from test.utils import mocked_count_method, mocked_query_method
+from unittest.mock import MagicMock
 
 import pandas as pd
 
-from kili.client import Kili
+from kili.queries.asset import QueriesAsset
 
-api_endpoint = os.getenv("KILI_API_ENDPOINT")
-kili = Kili(api_endpoint=api_endpoint)
+mocked_auth = MagicMock()
+mocked_auth.client.endpoint = "https://staging.cloud.kili-technology.com/api/label/v2/graphql"
```

Comment:
we should put a random string here right? otherwise it will call the staging server